CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ All notable changes to [bpmn-js](https://github.com/bpmn-io/bpmn-js) are documen
 
 ___Note:__ Yet to be released changes appear here._
 
+## 17.0.0
+
+* `DEPS`: update to `diagram-js@14.0.0`
```

Comment:
Let's add additional improvements here: 

```suggestion
* `FEAT`: add to selection through SHIFT ([bpmn-io/diagram-js#796](https://github.com/bpmn-io/diagram-js/pull/851), [#2053](https://github.com/bpmn-io/bpmn-js/issues/2053))
* `CHORE`: remove broken touch interaction ([bpmn-io/diagram-js#796](https://github.com/bpmn-io/diagram-js/issues/796))
* `DEPS`: update to `diagram-js@14.0.0`
```

Reply:
I will apply your suggestion on the release PR.